MATLAB/algorithms.prj

```diff
@@ -0,0 +1,211 @@
+<deployment-project plugin="plugin.ezdeploy" plugin-version="1.0">
+  <configuration file="C:\Users\Grzegorz\Source\Repos\MSI-dev\msi-divik\MATLAB\algorithms.prj" location="C:\Users\Grzegorz\Source\Repos\MSI-dev\msi-divik\MATLAB" name="algorithms" target="target.ezdeploy.library" target-name="Library Compiler">
```

Comment:
This should be generic. I was convinced MATLAB supports any portability by default.